events-drop-ins.html

```diff
@@ -12,6 +12,11 @@ <h2>Drop-in sessions</h2>
                 <div class="col-sm-4 col-lg-2 event-date">
                     <span class="event-date-month">SEP</span>
                     <span class="event-date-day">TBC</span>
+                    <p>
+                    <span class="event-date-start-time">7pm - </span>
```

Comment:
The times are really big all the way down this file. Can we use <smaller> or something?